src/main/java/Storage.java

```diff
@@ -0,0 +1,134 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.LinkedList;
+import java.util.Scanner;
+
+public class Storage {
+    private static String filePath = "";
+
+    /**
+     * To deals with loading tasks from the file and saving tasks in the file.
+     *
+     * @param filePath the location where the previous list of tasks is being saved.
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Load the list of task from collection of words/Strings.
+     *
+     * @return LinkedList of Task.
+     * @throws FileNotFoundException if no file detected.
+     * @throws DukeException if there is logic error or bad input.
+     * @throws IOException file corrupted.
+     */
+    public LinkedList<Task> load() {
+        File file = new File(filePath);
+        LinkedList<Task> loadedTasks = new LinkedList<>();
```

Comment:
I'm curious why you chose a `LinkedList` instead of an `ArrayList`.

Reply:
I am actually curious about this. The reason why I choose LinkedList rather than ArrayList is that LinkedList can do everything that ArrayList has and I think much faster. Why I think ArrayList is better when converting ArrayList to array is faster? What do you think?